tests/test_configuration_common.py

```diff
@@ -110,6 +115,34 @@ def create_and_test_config_from_and_save_pretrained_subfolder(self):
 
         self.parent.assertEqual(config_second.to_dict(), config_first.to_dict())
 
+    def create_and_test_config_from_and_save_pretrained_composite(self):
+        config = self.config_class(**self.inputs_dict)
```

Comment:
okay, as long as the attributes here are different than default we are testing good stuff